src/libais/decode_body.cpp

```diff
@@ -139,6 +139,8 @@ unique_ptr<AisMsg> CreateAisMsg8(const string &body, const int fill_bits) {
       switch (msg.fi) {
         case 22:
           return MakeUnique<libais::Ais8_367_22>(body.c_str(), fill_bits);
+        case 33:
```

Comment:
This PR has 8_367_23, 8_367_24, 25, and 33 all in one.  The others are missing here.  But let's back up and just do one message to start.  Maybe pick the simplest message.